.gitignore

```diff
@@ -8,6 +8,10 @@
 .AppleDouble
 .LSOverride
 
+# Google authorization files
+application_default_credentials.json
+service_account_key.json
+
```

Comment:
To prevent the checking-in of any credential files should someone put them in the repository.